rest-api/src/test/java/gaffer/rest/GraphFactoryTest.java

```diff
@@ -60,4 +61,16 @@ public void shouldReturnNullWhenCreateOpAuthoriserWithNoSystemPropertyPath() {
         // Then
```

Comment:
Not directed related to this particular pull request, but a test to show that an IllegalArgumentException is thrown if the Graph Factory Class name is invalid would be good